src/main/java/com/liferay/ide/idea/language/LiferayXmlFileReferenceFilterPattern.java

```diff
@@ -135,8 +134,10 @@ public boolean isClassAcceptable(Class hintClass) {
 					"portlet-decorator-thumbnail-path"));
 			put(
 				LiferayDefinitionsResourceProvider.XML_NAMESPACE_LIFERAY_LOOK_AND_FEEL_7_1_0,
-				_tagsMap.get(LiferayDefinitionsResourceProvider.XML_NAMESPACE_LIFERAY_LOOK_AND_FEEL_7_0_0));
-
+				Arrays.asList(
+					"root-path", "templates-path", "css-path", "images-path", "javascript-path",
```

Comment:
Do these entity names match the order in the DTD?  if not can you sort them accordingly?

Reply:
it already match by the dtd